setup.py

```diff
@@ -63,7 +64,7 @@
     # for example:
     # $ pip install -e .[dev,test]
     extras_require={
-        'scan': ['PyBluez==0.22'],
+        'scan': ['PyBluez==0.22'] if sys.platform.startswith("linux") else [],
```

Comment:
See first comment. Please only add special handling for FreeBSD and leave it otherwise as is.